mono/metadata/threads.h

```diff
@@ -12,6 +12,7 @@
 #ifndef _MONO_METADATA_THREADS_H_
 #define _MONO_METADATA_THREADS_H_
 
+#include <glib.h>
```

Comment:
as far as I know we don't want glib dependency in public header files

Reply:
Ok. Could I change gchar->char?